vue.config.js

```diff
@@ -123,6 +123,9 @@ module.exports = {
             declaration: true,
             noEmit: false,
             outDir: `${__dirname}/@types/${pluginName}`,
+            paths: {
```

Comment:
This updates https://github.com/matomo-org/matomo/blob/5.x-dev/tsconfig.json#L27-L29.

Reply:
Plugins being located outside of the Matomo root directory has been supported in core for quite a while now.